### ISSUE TITLE
Make the Parquet store CalculateSplitPoints classes run in parallel per Group

### ISSUE THREAD

**Issue by ac74475** [gchq/Gaffer#1352]
This will involve making the following classes Callable so they can be run on multiple threads:
- CalculateSplitPointsFromJavaRDD
- CalculateSplitPointsFromIterable

This will make the initial addElements operation slightly faster as it will not have to sequentially work out the split points for each Gaffer group.

**Comment by p013570** [comment]
Merged into develop.
